src/sdk/pynni/nni/smac_tuner/smac_tuner.py

```diff
@@ -261,3 +262,13 @@ def generate_multiple_parameters(self, parameter_id_list):
                 params.append(self.convert_loguniform_categorical(challenger.get_dictionary()))
                 cnt += 1
         return params
+
+    def feed_tuning_data(self, data):
```

Comment:
This one would not work, let me help you fix

Reply:
Will support in the following release.